src/xdocs/checks/misc/nocodeinfile.xml

```diff
@@ -45,12 +45,16 @@
           Content of the files:
         </p>
         <source>
-// single-line comment // violation
```

Comment:
Please remove line `Content of the files:` that is 3 lines above this line.
In final rendering it looks weird

Reply:
Removed....